.stylelintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "stylelint-config-recommended-scss",
+  "rules": {
+    "no-descending-specificity": null,
```

Comment:
This rule seems helpful. There a pattern we're following that contradicts this?

Reply:
No, we can enable it. I had originally asked if we wanted to enable this before fixing the issues since it's not autofixable. 🛠